app/views/admin/suppliers/_task.html.haml

```diff
@@ -6,7 +6,7 @@
       = task.period_date.to_s(:month_year)
   - if task.active_submission
     %td.govuk-table__cell.govuk-table__cell--numeric
-      - unless task.active_submission.total_spend.zero?
+      - unless task.active_submission.management_charge.zero?
```

Comment:
We should probably show the spend and management charge when either of them are non-zero(?)